{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-12-19T16:30:00-0400",
        "start": "2020-12-19T13:04:58-0400"
      },
      "geom": {
        "p": [
          "{rcpGzt~bK"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
